homeassistant/components/fan/__init__.py

```diff
@@ -45,6 +45,7 @@
 SPEED_OFF = 'off'
 SPEED_LOW = 'low'
 SPEED_MEDIUM = 'medium'
+SPEED_MED = 'med'
```

Comment:
Exactly what I would have said. I removed `SPEED_MED` because it's a platform specific constant which is confusing to maintain alongside of `SPEED_MEDIUM.`